### ISSUE TITLE
Shapes arent shown in Preview. 

### ISSUE THREAD

**Issue by thisisole** [runceel/plantumlpreview#20]
installed a new code-version... cant currently see the yellow shapes in the preview... only text and arrows are shown.

**Comment by runceel** [comment]
I think that this issue is VSCode's issue.
This plugin export svg then preview. Exported svg can preview Edge and Chrome, VSCode previewer can't preview svg(can't see the yellow shapes).

You can see yellow shape. follow:

```
@startuml
skinparam style strictuml

write your code!!
@enduml
```

Relative issue is #19.

**Comment by qjebbs** [comment]
SVG does not support all diagrams, and got issue above, so my suggestion is to switch to PNG, just like I did in https://github.com/qjebbs/vscode-plantuml.

key code here:
```typescript
let task = this.exporter.exportToBuffer(diagram, "png");
task.promise.then(
    result => {
        let b64 = result.toString('base64');
        this.image = `data:image/png;base64,${b64}`
        ...
    }
);
```

**Comment by runceel** [comment]
@qjebbs Thank you!!

I released new version. This version show correctry diagram.

**Comment by l4ces** [comment]
I have the latest version and preview only displays a white empty page with a tiny white square box at the upper left corner of the PlantUML Preview window.

VSCode version 1.9.1
Java paths and graphviz paths are set up correctly :-)

**Comment by runceel** [comment]
Thank you for your feedback.

This case is when export a unsaved document.
Please export after save document.

I released v0.2.1.
Please try again.(Added error message when export unsaved document.)
